db/migrations/mysql/000085_add_cloud_limits_archived.down.sql

```diff
@@ -0,0 +1,14 @@
+SET @preparedStatement = (SELECT IF(
+	EXISTS (
+		SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS
+		WHERE table_name = 'Teams'
+		AND table_schema = DATABASE()
+		AND column_name = 'CloudLimitsArchived'
+	),
+	'ALTER TABLE Teams DROP COLUMN IF EXISTS CloudLimitsArchived;',
```

Comment:
`TestUpAndDownMigrations/Should_be_reversible_for_mysql`

is failing because MySQL doesn't support `IF EXISTS`. That's why we do the convoluted EXISTS check a few lines above. Its a workaround for MySQL not supporting the `IF EXISTS` check in-line.

Reply:
Thanks